.github/workflows/build-and-deploy.yml

```diff
@@ -550,6 +550,33 @@ jobs:
           sha: ${{ github.sha }}
           slack-webhook: ${{ secrets.SLACK_WEBHOOK }}
 
+  deploy-aks-before-production:
+    name: Parallel deployment before production v2
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+    environment:
+      name: ${{ matrix.environment }}
+      url: ${{ steps.deploy_app_before_production_v2.outputs.deploy-url }}
+    needs: [lint, test]
+    runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
This would be safer to set it to false (even if it doesn't make much difference since the matrix runs in parallel)